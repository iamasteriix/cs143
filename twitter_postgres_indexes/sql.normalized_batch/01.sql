/*
 * Count the number of tweets that use #coronavirus
 */
SELECT count(distinct id_tweets)
FROM tweet_tags
WHERE tag='#coronavirus';
