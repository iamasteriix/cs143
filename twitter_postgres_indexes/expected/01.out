 count 
-------
  3841
(1 row)

